template/package.json

```diff
@@ -77,85 +77,89 @@
-    "electron": "^1.7.5",
-    "electron-debug": "^1.4.0",
-    "electron-devtools-installer": "^2.2.0",
+    "electron": "^2.0.2",
+    "electron-debug": "^1.5.0",
+    "electron-devtools-installer": "^2.2.4",
     {{#if_eq builder 'packager'}}
-    "electron-packager": "^8.5.0",
-    "electron-rebuild": "^1.1.3",
+    "electron-packager": "^12.1.0",
+    "electron-rebuild": "^1.7.3",
     {{else}}
-    "electron-builder": "^19.19.1",
+    "electron-builder": "^20.15.1",
     {{/if_eq}}
     {{#eslint}}
-    "babel-eslint": "^7.2.3",
-    "eslint": "^4.4.1",
-    "eslint-friendly-formatter": "^3.0.0",
-    "eslint-loader": "^1.9.0",
-    "eslint-plugin-html": "^3.1.1",
+    "babel-eslint": "^8.2.3",
+    "eslint": "^4.19.1",
+    "eslint-friendly-formatter": "^4.0.1",
+    "eslint-loader": "^2.0.0",
+    "eslint-plugin-html": "^4.0.3",
     {{#if_eq eslintConfig 'standard'}}
-    "eslint-config-standard": "^10.2.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-node": "^5.1.1",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^3.0.1",
+    "eslint-config-standard": "^11.0.0",
+    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-promise": "^3.8.0",
+    "eslint-plugin-standard": "^3.1.0",
     {{/if_eq}}
     {{#if_eq eslintConfig 'airbnb'}}
-    "eslint-config-airbnb-base": "^11.2.0",
-    "eslint-import-resolver-webpack": "^0.8.1",
-    "eslint-plugin-import": "^2.2.0",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-import-resolver-webpack": "^0.10.0",
+    "eslint-plugin-import": "^2.12.0",
     {{/if_eq}}
     {{/eslint}}
-    "extract-text-webpack-plugin": "^3.0.0",
-    "file-loader": "^0.11.2",
-    "html-webpack-plugin": "^2.30.1",
+    "mini-css-extract-plugin": "0.4.0",
+    "file-loader": "^1.1.11",
+    "html-webpack-plugin": "^3.2.0",
     {{#if unit}}
-    "inject-loader": "^3.0.0",
-    "karma": "^1.3.0",
+    "inject-loader": "^4.0.1",
+    "karma": "^2.0.2",
     "karma-chai": "^0.1.0",
-    "karma-coverage": "^1.1.1",
-    "karma-electron": "^5.1.1",
-    "karma-mocha": "^1.2.0",
+    "karma-coverage": "^1.1.2",
+    "karma-electron": "^6.0.0",
+    "karma-mocha": "^1.3.0",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-spec-reporter": "^0.0.31",
-    "karma-webpack": "^2.0.1",
+    "karma-spec-reporter": "^0.0.32",
+    "karma-webpack": "^3.0.0",
     "webpack-merge": "^4.1.0",
     {{/if}}
     {{#if e2e}}
-    "require-dir": "^0.3.0",
-    "spectron": "^3.7.1",
+    "require-dir": "^1.0.0",
+    "spectron": "^3.8.0",
     {{/if}}
     {{#testing unit e2e}}
-    "babel-plugin-istanbul": "^4.1.1",
-    "chai": "^4.0.0",
-    "mocha": "^3.0.2",
+    "babel-plugin-istanbul": "^4.1.6",
+    "chai": "^4.1.2",
+    "mocha": "^5.2.0",
     {{/testing}}
     "multispinner": "^0.2.1",
     "node-loader": "^0.6.0",
-    "style-loader": "^0.18.2",
-    "url-loader": "^0.5.9",
+    "node-sass": "^4.9.0",
+    "sass-loader": "^7.0.1",
```

Comment:
Let's remove `node-sass` and `sass-loader`. No need to enforce this upon all developers.